src/irmin-pack/unix/ext.ml

```diff
@@ -284,7 +284,7 @@ module Maker (Config : Conf.S) = struct
           let start_exn ?(unlink = true) ~use_auto_finalisation t commit_key =
             match t.running_gc with
             | Some _ ->
-                [%log.info "Repo is alreadying running GC. Skipping."];
```

Comment:
What was the reason for making this debug? I could see this log being helpful in octez logs if somehow the GC gets stuck. It also corresponds with the `log.info` for starting a GC.

Reply:
This was flooding my replays at some point but I think it can be added back

Reply:
Are you planning to change this back to info?